pandas/core/groupby.py

```diff
@@ -2742,74 +2741,124 @@ def _convert_grouper(axis, grouper):
-    for _def_str in _whitelist_method_generator(Series,
-                                                _series_apply_whitelist):
-        exec(_def_str)
+
+    @Appender(Series.all.__doc__)
+    def all(self, axis=None, bool_only=None, skipna=None, level=None,
+            **kwargs):
+        f = self.__getattr__('all')
+        return f(axis=axis, bool_only=bool_only, skipna=skipna, level=level)
+
+    @property
+    def dtype(self):
+        """
+         return the dtype object of the underlying data
+        """
+        return self.__getattr__('dtype')
+
+    @Appender(Series.idxmax.__doc__)
+    def idxmax(self, axis=None, skipna=True, *args, **kwargs):
+        f = self.__getattr__('idxmax')
+        return f(axis=axis, skipna=skipna)
+
+    @Appender(Series.rank.__doc__)
+    def rank(self, axis=0, method='average', numeric_only=None,
+             na_option='keep', ascending=True, pct=False):
+        f = self.__getattr__('rank')
+        return f(axis=axis, method=method, numeric_only=numeric_only,
+                 na_option=na_option, ascending=ascending, pct=pct)
+
+    @Appender(Series.diff.__doc__)
+    def diff(self, periods=1):
+        f = self.__getattr__('diff')
+        return f(periods=periods)
+
+    @Appender(Series.any.__doc__)
+    def any(self, axis=None, bool_only=None, skipna=None, level=None,
+            **kwargs):
+        f = self.__getattr__('any')
+        return f(axis=axis, bool_only=bool_only, skipna=skipna, level=level)
+
+    @Appender(Series.nsmallest.__doc__)
+    def nsmallest(self, n=5, keep='first'):
+        f = self.__getattr__('nsmallest')
+        return f(n=n, keep=keep)
+
+    @Appender(Series.quantile.__doc__)
+    def quantile(self, q=0.5, interpolation='linear'):
+        f = self.__getattr__('quantile')
+        return f(q=q, interpolation=interpolation)
+
+    @Appender(Series.hist.__doc__)
+    def hist(self, by=None, ax=None, grid=True, xlabelsize=None, xrot=None,
+             ylabelsize=None, yrot=None, figsize=None, bins=10, **kwds):
+        f = self.__getattr__('hist')
+        return f(by=by, ax=ax, grid=grid, xlabelsize=xlabelsize, xrot=xrot,
+                 ylabelsize=ylabelsize, yrot=yrot, figsize=figsize, bins=bins)
+
+    @Appender(Series.take.__doc__)
+    def take(self, indices, axis=0, convert=True, is_copy=False, **kwargs):
+        f = self.__getattr__('take')
+        return f(indices=indices, axis=axis, convert=convert, is_copy=is_copy)
+
+    @Appender(Series.mad.__doc__)
+    def mad(self, axis=None, skipna=None, level=None):
+        f = self.__getattr__('mad')
+        return f(axis=axis, skipna=skipna, level=level)
+
+    @Appender(Series.corr.__doc__)
+    def corr(self, other, method='pearson', min_periods=None):
+        f = self.__getattr__('corr')
+        return f(other=other, method=method, min_periods=min_periods)
+
+    @Appender(Series.fillna.__doc__)
+    def fillna(self, value=None, method=None, axis=None, inplace=False,
```

Comment:
many of these keywords need to be eliminated. e.g. ``axis`` and ``inplace``, they don't make sense on groupby objects.